OpenRA.Mods.Common/Widgets/Logic/Settings/SettingsUtils.cs

```diff
@@ -49,6 +52,26 @@ public static void BindIntSliderPref(Widget parent, string id, object group, str
 			ss.OnChange += x => field.SetValue(group, (int)x);
 		}
 
+		public static void BindColorWidget(Widget panel, WorldRenderer worldRenderer, ColorPickerManagerInfo manager, Func<bool> isDisabled, string stance, object group, string pref)
```

Comment:
Both `object` and "pref" bother me here. The `object` one is because the colors are in different types of settings, as commented above, and the "pref" should just be "preference", but best call it "fieldName".

Reply:
I just followed the convention

Reply:
renamed a few things that weren't to convention